_i18n/es.yml

```diff
@@ -341,3 +341,10 @@ latest-publications:
   posts: Últimos posts
 author:
   publications-title: Publicaciones
+thanks:
+  get-back-soon: Agradecemos que se haya tomado el tiempo de escribirnos. Nos pondremos en contacto contigo pronto.
```

Comment:
You use third person and second in the two sentences. It should be one or the other but not mix both. Ex
Agradecemos que te hayas tomado el tiempo de escribirnos. Nos pondremos en contacto contigo pronto.

or

Agradecemos que se haya tomado el tiempo de escribirnos. Nos pondremos en contacto con usted pronto.